src/runtime/entitlements-manager.js

```diff
@@ -60,6 +61,15 @@ export class EntitlementsManager {
 
     /** @private @const {!./storage.Storage} */
     this.storage_ = deps.storage();
+
+    /** @private @const {string} */
+    this.toastSrc_ = feUrl('/toastiframe');
```

Comment:
These are rarely used. I recommend you just build them right before showing the toast - no values to have them as state params.

Reply:
Moved